localstack/services/cloudformation/resource_provider.py

```diff
@@ -60,7 +60,8 @@
     "AWS::Route53::HealthCheck": "ResourceProvider",
     "AWS::Route53::RecordSet": "ResourceProvider",
     "AWS::SNS::Topic": "ResourceProvider",
-    "AWS::Kinesis::Stream": "ResourceProvider"
+    "AWS::Kinesis::Stream": "ResourceProvider",
+    "AWS::KinesisFirehose::DeliveryStream": "ResourceProvider"
```

Comment:
super-minor: it might be worth adding a trailing comma to the last item in the list so there is only a 1-line change per PR